inference-engine/samples/benchmark_app/CMakeLists.txt

```diff
@@ -8,5 +8,5 @@ file (GLOB HDR ${CMAKE_CURRENT_SOURCE_DIR}/*.hpp)
 ie_add_sample(NAME benchmark_app
               SOURCES ${SRC}
               HEADERS ${HDR}
-              DEPENDENCIES format_reader ie_samples_utils
+              DEPENDENCIES format_reader ie_samples_utils ngraph::frontend_manager
```

Comment:
It is really strange that we had to add ngraph::frontend_manager dependency here and in helo_classification sample. Not clear why other samples don't need this dependency as they also use IE Core objects with ReadNetwork/LoadNetwork methods

Reply:
Totally agree